src/js/msp/MSPConnector.js

```diff
@@ -4,11 +4,12 @@ import { i18n } from "../localization";
 import GUI from "../gui";
 import MSP from "../msp";
 import FC from "../fc";
+import { serialShim } from "../serial_shim";
 import MSPCodes from "./MSPCodes";
 import CONFIGURATOR from "../data_storage";
-import serial from "../serial";
 import { gui_log } from "../gui_log";
 
+const serial = serialShim();
```

Comment:
Already removed the shim here in `#3996`.

Same in other places as serial.js is removed in this PR we can just import serial from serialWeb in all places.